integration-tests/proxy_test.go

```diff
@@ -130,12 +134,28 @@ func TestProxyEgressUDP(t *testing.T) {
 	ctxTimeout, clientCancel := context.WithTimeout(ctx, 10*time.Second)
 	defer clientCancel()
 	success, err := util.CheckPeriodically(ctxTimeout, time.Second, func() (bool, error) {
-		output, err := helper.containerExec(ctx, node2, []string{"udping", "127.0.0.1", "4242"})
-		if err != nil {
-			helper.Logf("Retrying udp client for up to 10 seconds while waiting for peering to finish: %v -- %s", err, output)
-			return false, nil
+		type target struct {
+			IP   string
+			Port string
+		}
+		targets := []target{
+			// v4 client, v4 server
+			{IP: "127.0.0.1", Port: "4242"},
+			// v4 client, v6 server
+			{IP: "127.0.0.1", Port: "4243"},
+			// v6 client, v4 server
+			{IP: "::1", Port: "4242"},
+			// v6 client, v6 server
+			{IP: "::1", Port: "4243"},
+		}
```

Comment:
```suggestion
                targets := []struct {
		    IP   string
		    Port string
	        }{
		    // v4 client, v4 server
		    {IP: "127.0.0.1", Port: "4242"},
		    // v4 client, v6 server
		    {IP: "127.0.0.1", Port: "4243"},
		    // v6 client, v4 server
		    {IP: "::1", Port: "4242"},
		    // v6 client, v6 server
		    {IP: "::1", Port: "4243"},
                }
```

nit: You can use an anonymous struct here

Reply:
Oh, nice. Yeah that’s perfect. I’ll change.

Reply:
done!